common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -237,6 +237,24 @@ def handle_ajax(self, dispatch, data):  # TODO: bounds checking
             return json.dumps({
                 'complete': complete
             })
+        elif dispatch == 'metadata':
+            if self.category != 'sequential':
```

Comment:
If the idea is prevent subclasses of `SequenceModule` (like the root course descriptor) from answering here, please make an explicit check against the class–it's not common, but we do accept a couple aliases for `sequential` in the OLX. Also, please add a comment explaining why you're doing this.

Reply:
@ormsbee That was the intention, because the endpoint will fail if requested from the course descriptor. How about, instead of checking for classes, we just let that behavior be undefined? It wouldn't return meaningful metadata for the course, anyway.